config/locales/mailers/en.yml

```diff
@@ -27,224 +30,331 @@ en:
         left_kudos:
           one: "%{givers_list} left kudos on %{commentable_link}."
           other: "%{givers_list} left kudos on %{commentable_link}."
+      you_have: "[%{app_name}] You've got kudos!"
```

Comment:
The other PR has some HTML tags that get added in the html left_kudos keys. https://github.com/otwcode/otwarchive/pull/3952/files#r550414843